pkg/component/component.go

```diff
@@ -394,23 +393,12 @@ func CreateComponent(client *occlient.Client, componentConfig config.LocalConfig
 	envVarsList := componentConfig.GetEnvVars()
 	addDebugPortToEnv(&envVarsList, componentConfig)
 
-	// create and get the storage to be created/mounted during the component creation
-	storageList, err := getStorageFromConfig(&componentConfig)
-	if err != nil {
-		return err
-	}
-	storageToBeMounted, _, err := storage.S2iPush(client, storageList, componentConfig.GetName(), componentConfig.GetApplication(), false)
-	if err != nil {
-		return err
-	}
-
 	log.Successf("Initializing component")
 	createArgs := occlient.CreateArgs{
-		Name:               cmpName,
-		ImageName:          cmpType,
-		ApplicationName:    appName,
-		EnvVars:            envVarsList.ToStringSlice(),
-		StorageToBeMounted: storageToBeMounted,
```

Comment:
I found a few references for `.StorageToBeMounted` in occlient.go and occlient_test.go, can you remove them as well?

Reply:
It will removed in one of the later PRs.